ruby/main.rb

```diff
@@ -30,6 +32,10 @@
   "Sentry Ruby Service says Hello - turn me into a microservice that powers Invoicing, Trucking, or DriverFind"
 end
 
+# get"/products" do
```

Comment:
@bfpepper87 let's get rid of this since I think we didn't finish developing this. Maybe you should switch git branches if you're switching to a new demo feature, so you don't confuse code. OPTIONS demo task is different than the ruby stuff you were working on.